bloom-housing/bloom
# feat: add confirm modal when copying a preference

# Pull Request Template

## Issue Overview

This PR addresses #3029


## Description

Adds a new confirmation modal when copying a preference.

## How Can This Be Tested/Reviewed?

Copy some preferences - compare the modal to the designs, and ensure the new listing prepends "Copy of".

Note: Offline, we removed the "clone" language and are using "copy" language.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
